Documentation/Contributing/rook-test-framework.md

```diff
@@ -56,7 +56,7 @@ See [environment.go](/tests/framework/installer/environment.go) for the availabl
 Set the following variables:
 
 ```console
-export TEST_HELM_PATH=/tmp/rook-tests-scripts-helm/linux-amd64/helm
+export TEST_HELM_PATH=/tmp/rook-tests-scripts-helm/helm
```

Comment:
After running `helm.sh up`, I have:
```
$ ls /tmp/rook-tests-scripts-helm/
helm  helm.tar.gz  LICENSE  README.md
```

I suspect the layout in the Helm archive changed (again). Last time appears to have been in 2020, when `--strip-components 1` was introduced in 27c714652633a04acae217b0069ff208814a077a. Maybe this needs to be reverted now instead of changing the path in the docs.